internal/pkg/cli/diff.go

```diff
@@ -44,7 +44,7 @@ func diffCommand(root *rootCommand) *cobra.Command {
 					for _, result := range diffResults.Results {
 						if result.State != diff.ResultEqual {
 							// Message
-							msg := fmt.Sprintf("%s %s %s", result.Mark(), result.Kind().Abbr, result.RelativePath())
+							msg := fmt.Sprintf("%s %s %s", result.Mark(), result.Kind().Abbr, result.Path())
```

Comment:
Premenoval som metodu `RelativePath` -> `Path`, ... je to dokoncenie z minula, ... kedze odkedy je virtual filesystem, tak su vsetky cesty k objektom relativne.